test/unit/cameras/PerspectiveCamera.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * @author simonThiele / https://github.com/simonThiele
+ */
+
+module( "PerspectiveCamera" );
+
+test( "updateProjectionMatrix", function() {
+	var near = 1,
+			far = 3,
+			bottom = -1,
+			top = 1,
```

Comment:
Sorry for all the questions, but I am trying to understand what you are computing yourself, and what you are testing.

If you change the `fov`, then `top` must change, too. `top` depends on `fov`, you can't set it as an input.

Also, you cannot use the code to test itself. That is the thing that concerns me the most.

I would prefer you hand-calculate the answer, and then compare against your hand-calculated values. And try to avoid trivial test cases.

Thank you for doing this. It is important. I just want to make sure the tests are meaningful, and that requires some thought.